examples/autoscale/main.tf

```diff
@@ -170,77 +146,51 @@ module "autoscale" {
     dhcp-send-client-id         = "yes"
     dhcp-accept-server-hostname = "yes"
     dhcp-accept-server-domain   = "yes"
-    dns-primary                 = "8.8.8.8"
+    dns-primary                 = "169.254.169.254" # Google DNS required to deliver PAN-OS metrics to Cloud Monitoring
     dns-secondary               = "4.2.2.2"
   }
 
-  # Example of full bootstrap with Google storage bucket.
-  /*
-  metadata = {
-    mgmt-interface-swap                  = "enable"
-    vmseries-bootstrap-gce-storagebucket = "my-google-bootstrap-bucket"
-    serial-port-enable                   = true
-    ssh-keys                             = "~/.ssh/vmseries-ssh-key.pub"
-  }
-  */
-  named_ports = [
-    {
-      name = "http"
-      port = "80"
-    },
+  depends_on = [
```

Comment:
Do we need this dependency here ? I'm guessing it would be required for bootstrap but maybe it works without it ?

Reply:
Correct, this guarantees that once deployed, vmseries instances can reach the bootstrap bucket.